chains/chain-bridge-hub-kusama/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "bp-bridge-hub-kusama"
+description = "Primitives of BridgeHubKusama parachain runtime."
+version = "0.6.0"
+authors.workspace = true
+edition.workspace = true
+license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
+repository.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+# Bridge Dependencies
+
+bp-bridge-hub-cumulus = { git = "https://github.com/paritytech/polkadot-sdk.git", branch="master"}
```

Comment:
@serban300 maybe we could also switch to workspace dependencies, wdyt?

Reply:
Yes, I was also thinking about this. Would be nice to do this. But let's do it in a separate PR